src/script.js

```diff
@@ -71,8 +85,10 @@ if (localStorage.getItem('itemList')) {
 
 addButton.addEventListener('click', () => {
   if (inputValue.value !== '') {
-    // eslint-disable-next-line no-new
-    new Item(inputValue.value);
+    const newItem = new Item(inputValue.value);
+    itemList.push(newItem);
+    /* eslint-disable no-console */
```

Comment:
As liners help us to improve the code quality and it's mandatory on Microverse.

- Kindly remove this line to enable linters and fix possible errors